fv3gfs-physics/tests/savepoint/translate/translate_fv_update_phys.py

```diff
@@ -376,12 +374,10 @@ def compute_parallel(self, inputs, communicator):
         out["qsnow"] = state.qsnow[self.grid.slice_dict(ds)]
         out["qgraupel"] = state.qgraupel[self.grid.slice_dict(ds)]
         out["pt"] = state.pt[self.grid.slice_dict(ds)]
-        state.u.synchronize()
-        state.v.synchronize()
         state.ua.synchronize()
         state.va.synchronize()
```

Comment:
do you know why we do or do not need the synchronize for u vs ua?

Reply:
I think this one is because we're not using `slice_output`, which calls synchronize with certain backend. Since here the output is prepared manually, we call synchronize explicitly for the variables that were halo exchanged.